virtool/groups/oas.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+from pydantic import BaseModel, Field
+
+
+class SamplePermissions(BaseModel):
```

Comment:
Why does this start with `Sample`? Change name to `EditGroupSchema` as noted in`api.py`.

With this schema, we need to make sure that the request doesn't set some permissions to `None` in the database. In other words, if our request body is `{modify_hmm: true}`, are all the other permissions going to stay as they are or be set to `None`?

Reply:
Using permissions = data.permissions.dict(exclude_unset=True) will create a dict that only contains the permissions we want to update, excluding the rest.